recipe/run_test.py

```diff
@@ -14,4 +14,7 @@
 import numpy.linalg.lapack_lite
 import numpy.random.mtrand
 
+assert numpy.__config__.get_info("openblas") != {}
```

Comment:
Make sure we are getting `openblas`. Though this is going to need to be addressed in a more friendly manner for `noblas`.